src/parallel/LookaheadSplitter.cc

```diff
@@ -34,9 +34,8 @@ bool LookaheadSplitter::createSplitLookahead(LASplitNode & n)
             // In cases where the LA solver couldn't propagate due to
             // literal being already assigned, the literal may be
             // duplicated.  Do not report duplicates.
-            tmp.push(l);
```

Comment:
There is now an unused `vec<Lit> tmp` on line 31.
I wonder if the compiler will warn about it.

Reply:
It doesn't, but I'll fix it.